README.md

```diff
@@ -21,10 +21,14 @@ This repository contains Docker configuration aimed at Moodle developers and tes
 
 ```bash
 # Set up path to Moodle code
-export MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
+export MOODLE_DOCKER_WWWROOT=./moodle
```

Comment:
I'd keep the `/path/to/moodle/code`, sorry!

Reply:
/path/to/moodle documented at

https://github.com/moodlehq/moodle-docker/pull/286/commits/1bd5c9d098cdfeb819417b391c8f303c1a374e82